subiquity/models/filesystem.py

```diff
@@ -35,7 +35,10 @@
 
 log = logging.getLogger('subiquity.models.filesystem')
 
-MiB = 1024 * 1024
+
+kiB = 1 << 10
+MiB = 1 << 20
+GiB = 1 << 30
```

Comment:
I mean, fine, but not uses anywhere in this branch?

Reply:
GiB was used in the tests.  I wanted to have them predefined to make it easier for later uset.